docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+    }
+
+    void SetUnsetFeedbackAudioFormatChangedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged += feedbackAudioFormatChanged;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged -= feedbackAudioFormatChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set a callback function to be called when engine sends audio streaming for TTS feedback:
+    ```
+    /// callback
+    void feedbackStreaming(object sender, FeedbackStreamingEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetFeedbackStreamingCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackStreaming += feedbackStreaming;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackStreaming -= feedbackStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set TTS streaming callback function is called when the vc client sends audio streaming for TTS feedback:
+    ```
+    /// callback
+    void vcTtsStreaming(object sender, VcTtsStreamingEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetVcTtsStreamingCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.VcTtsStreaming += vcTtsStreaming;
+
+            /// Unset the callback
+            VoiceControlManagerClient.VcTtsStreaming -= vcTtsStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="start_and_stop_recording"></a>
+## Start and stop recording
+
+You can start, stop, or cancel in voice control manager:
+
+-   To start recording, use the `Start()` method with `exclusiveCommandOption` as the parameter.
+    The connected voice control service starts recording and the voice control manager state is changed to `Recording`.
+    If the parameter `exclusiveCommandOption` value is true, voice control service recognizes in only exclusive commands.
+    This method must be called when the voice control manager is in the `Ready` state:
+
+    > **Note**   
+    >
+    > If the `Start()` method fails, check the error code and take appropriate action.
+    ```
+    void StartManager(bool exclusiveCommandOption)
+    {
+        try
+        {
+            VoiceControlManagerClient.Start(exclusiveCommandOption);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   To stop recording, use the `Stop()` method.
+    The recording stops and the voice control manager state is changed to `Processing`.
+    When the recognition command result has been processed, the `RecognitionResult` event triggers, and the state changes back to `Ready`.
```

Comment:
Referencing https://github.com/Samsung/tizen-docs/pull/971#discussion_r370163104

>@sungraejo Suggested rephrase:
>
>When the recognition command result is processed, the RecognitionResult event triggers, and the state changes back to Ready.
>
>Kindly verify the technical meaning before incorporating the suggestion.

Reply:
@mijin85cho 
Done,
```
When the recognition command result has been processed, the `RecognitionResult` event triggers, and the state changes back to `Ready`.
```
to
```
When the recognition command result is processed, the `RecognitionResult` event triggers, and the state changes back to `Ready`.
```